CHANGELOG.md

```diff
@@ -7,7 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### `snforge_scarb_plugin`
+### Forge
+
+#### Added
+
+- Requirements validation during `snforge` runtime
+- `snforge validate-requirements` command
```

Comment:
Alternatively, it could be named `snforge doctor`. This is pretty popular pattern to use this word for dependencies checks (ref `flutter doctor`, `npx @react-native-community/cli doctor`).

Reply:
Changing to check-requirements